charlesinto/iReporter
# 162592359 Delete intervention

#### What does this PR do?

- Delete intervention record

#### Description of Task to be completed?
- implement JWTauthentication
- Verify user token
- User should be able to Delete a record

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `DELETE` request to `http://localhost:7000/api/v1/interventions/:id`
- You should receive a successful response

#### Any background context you want to provide?
Request should be structure like:

`{comment: "string"}`

`

#### What are the relevant pivotal tracker stories?
[#162592359](https://www.pivotaltracker.com/story/show/162592359)

#### Screenshots (if appropriate)






#### Questions:
N/A
